src/contracts/UpgradeRegressionManager.sol

```diff
@@ -353,6 +355,12 @@ contract UpgradeRegressionManager is IUpgradeRegressionManager {
     return uint256(keccak256(abi.encode(_targets, _values, _calldatas, _description)));
   }
 
+  /// @notice Returns the current state of a proposed rollback by its ID.
+  /// @param _rollbackId The ID of the rollback to check.
+  function state(uint256 _rollbackId) public view returns (IGovernor.ProposalState) {
```

Comment:
This function can be marked as `external` I believe

Reply:
Good catch :D